docker/Dockerfile

```diff
@@ -0,0 +1,25 @@
+FROM ubuntu:24.04
```

Comment:
Could you try the following:
https://github.com/zulu-openjdk/zulu-openjdk/blob/master/ubuntu/21-latest/Dockerfile

Reply:
Thanks for the suggestion Carl, we have updated the Dockerfile in this [commit](https://github.com/ikmdev/komet/pull/93/commits/7b80c7cc6428b7b7df971f2dce339663d36cc8dd).